docs/README_RECON.md

```diff
@@ -87,7 +98,28 @@ class Table:
     transformations: list[Transformation] | None = None
     thresholds: list[Thresholds] | None = None
     filters: Filters | None = None
-```
+</pre>
+</td>
+<td>
+<pre lang="json">
+{
+  "source_name": "product",
+  "target_name": "product",
```

Comment:
Please update all the actual values with <TO-BE-FILLED> elements. Here we are just providing a reference on how you can update the structure.